Dockerfile

```diff
@@ -2,25 +2,23 @@ FROM dwolla/jenkins-agent-core
 MAINTAINER Dwolla Dev <dev+jenkins-sbt@dwolla.com>
 LABEL org.label-schema.vcs-url="https://github.com/Dwolla/jenkins-agent-docker-sbt"
 
-ENV SBT_VERSION=0.13.15 \
+ENV SBT_VERSION=1.0.3 \
     SBT_HOME=/usr/local/sbt
 ENV PATH=${SBT_HOME}/bin:${PATH}
 
 USER root
 
 COPY fake-project $JENKINS_HOME/fake-project
 
-# (There is a bug in the sbt 0.13.13 packaging, so add a symlink in that specific case.)
-RUN apk add --no-cache git && \
-    curl -sL /tmp/sbt-${SBT_VERSION}.tgz "https://dl.bintray.com/sbt/native-packages/sbt/${SBT_VERSION}/sbt-${SBT_VERSION}.tgz" | \
+RUN apk add --no-cache git bc && \
```

Comment:
Had to add `bc` because of https://github.com/sbt/sbt-launcher-package/pull/182. 🙄 but it's only a few kb so 🤷🏻‍♂️